zookeeper/3.0/Dockerfile

```
@@ -35,7 +35,7 @@ RUN mkdir $ZK_HOME/data && \
 # Download and install Zookeeper
 #
 #RUN curl -fSL -o /tmp/zookeeper.tar.gz $(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 | sed -rn 's/.*"preferred":.*"(.*)"/\1/p')$ZK_URL_PATH || curl -fSL -o /tmp/zookeeper.tgz https://archive.apache.org/dist/$ZK_URL_PATH
-RUN curl -fSL -o /tmp/zookeeper.tar.gz https://archive.apache.org/dist/$ZK_URL_PATH
+RUN curl -fSL -o /tmp/zookeeper.tar.gz https://dlcdn.apache.org/dist/$ZK_URL_PATH
```

@vjuranek 
```suggestion
RUN curl -fSL -o /tmp/zookeeper.tar.gz https://dlcdn.apache.org/$ZK_URL_PATH
```
See https://github.com/debezium/container-images/commit/1962588836501c8365d65169e4715210e7662531#diff-03568d84dde3596c3880e87dbb1359543856ad94345f13165306873f4e731b78L38

---

OMG, thanks @jpechane